react/AppSections/categories.js

```diff
@@ -48,7 +48,7 @@ export const sorter = (categoryA, categoryB) => {
     (categoryB.value === 'all' && 1) ||
     (categoryA.value === 'others' && 1) ||
     (categoryB.value === 'others' && -1) ||
-    categoryA.label > categoryB.label
+    categoryA.label.localeCompare(categoryB.label)
```

Comment:
??? this generates a change in Argos. I wonder if this is necessar
<img width="640" alt="Capture d’écran 2022-01-03 à 14 07 31" src="https://user-images.githubusercontent.com/8363334/147934017-a48212d2-6947-4ea4-98ac-af210a52d934.png">
y

Reply:
https://github.com/cozy/cozy-ui/pull/1835/files#r777467457